File: src/Point.java
//import org.json.*;

public class Point {
    private double x;
    private double y;
    private double z;
    private Device device;


    public Point(double x, double y, double z, Device device) {
        this.x = x;
        this.y = y;
        this.z = z;
        this.device = device;
    }

    public double getX() {
        return this.x;
    }

    public void setX(double x) {
        this.x = x;
    }

    public double getY() {
        return this.y;
    }

    public void setY(double y) {
        this.y = y;
    }

    public double getZ() {
        return this.z;
    }

    public void setZ(double z) {
        this.z = z;
    }

    public Device getDevice() {
        return this.device;
    }

    public void setDevice(Device device) {
        this.device = device;
    }

}

File: Python/Point.py
class Point:
    def __init__(self, x, y, z, device):
        self.x = x
        self.y = y
        self.z = z
        self.device = device

    def get_x(self):
        return self.x

    def set_x(self, x):
        self.x = x

    def get_y(self):
        return self.y

    def set_y(self, y):
        self.y = y

    def get_z(self):
        return self.z

    def set_z(self, z):
        self.z = z

    def get_device(self):
        return self.device

    def set_device(self, device):
        self.device = device



